multi-node-k8s.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+set -e -o pipefail
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//')"
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible \
+	golang
+
+# install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/linux/amd64/kubectl
+mv kubectl /usr/bin/kubectl
+chmod +x /usr/bin/kubectl
+
+# Vagrant needs libvirtd running
+systemctl start libvirtd
+
+# Log the virsh capabilites so that we know the
+# environment in case something goes wrong.
+virsh capabilities
+
+# TODO: this is not the right way to install vagrant on CentOS, but SCL only
+# provides vagrant 1.x and we need >= 2.2
+yum -y install https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
+yum -y install gcc libvirt-devel
+
+vagrant plugin install vagrant-libvirt
+
+# setup the kubernes cluster
+git clone https://github.com/galexrt/k8s-vagrant-multi-node
```

Comment:
checkout released version to avoid regression in master

Reply:
master should be stable? we can report issues if there is a problem with it.
When a problem occurs, and is not fixed soon enough, we can always add an option to fetch an older version